lms/envs/aws.py

```diff
@@ -171,6 +171,10 @@
 for feature, value in ENV_FEATURES.items():
     FEATURES[feature] = value
 
+# Backward compatibility for deprecated feature names
+if 'ENABLE_S3_GRADE_DOWNLOADS' in FEATURES:
+    FEATURES['ENABLE_GRADE_DOWNLOADS'] = FEATURES['ENABLE_S3_GRADE_DOWNLOADS']
```

Comment:
But check to see what other systems have done. Also note that this is likely to affect the main configurations and you may need to make a pull request to the configuration repo if you haven't already-- I'm still looking through your PRs.